README.md

```diff
@@ -1,3 +1,39 @@
 # Ada Developers Academy Jump Start
 
-This repository contains the materials presented during the Ada Developers Academy jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+This repository contains the materials presented during the [Ada Developers Academy](adadevelopersacademy.org) jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+
+## Learning Goals
+_Problem Solving_  
+Breaking down complex problems is a fundamental skill as a software engineer. While we will be doing this for years to come, we want to give you an opportunity to explore problem solving in programatic ways an exploring topics like:
+- input
+- output
+- algorithms
+- variable substitution
+
+_Learning Style_  
+Whether you have gone through a traditional post-secondary program recently or not, coming to the Ada classroom can be an adjustment for many folks. It is good to get a gauge on your learning style as soon as possible to be able to jump right in on day one and absorb the material in a way that works well for you.
```

Comment:
My thought was that is what we want it to be in the future, but not what it is right now so yes, we would definitely re-work when we open to the public.